src/containers/MyNdla/components/UserProfileTag.tsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import styled from '@emotion/styled';
+import { fonts, colors, spacing } from '@ndla/core';
+import SafeLink from '@ndla/safelink';
+import ModeratorTag from './ModeratorTag';
+import Avatar from './Avatar';
+import { useArenaUser } from '../arenaQueries';
+
+type UserProfileTagProps = {
+  displayName: string | undefined;
+  username: string | undefined;
+  affiliation: string | undefined;
+};
+
+const Name = styled.div`
+  text-decoration: underline;
+  ${fonts.sizes('22px', '30px')}
+  font-weight: ${fonts.weight.bold};
+`;
+
+const UserProfileTagContainer = styled(SafeLink)`
+  display: flex;
+  gap: ${spacing.normal};
+  color: ${colors.text.primary};
+  height: fit-content;
+  width: fit-content;
+  text-decoration: none;
+  box-shadow: none;
+  &:hover {
+    ${Name} {
```

Comment:
Dette ser ikkje heilt korrekt ut.

Reply:
visuelt eller kodemessig? det ser kanskje litt rart ut siden det skal ha en underline på navnet til brukeren til vanlig, også skal underline forsvinne når denne komponenten hovres. da måtte jeg selecte Name som er under UserProfileTagContainer

Reply:
Har berre ikkje sett denne måten å gjøre det på.
Det at du definerer en styled div med navn Name gjør at du kan adressere generert klasse i css slik. Fiffig!

Reply:
Ok, det var rart for det virker som det er ganske vanlig å bruke component selectors. Jeg kan godt endre, men da må du nesten komme med en litt mer konkret forslag for jeg fikk ikke det helt til ved bruk av data-attributt.